Tools/CISupport/ews-build/steps.py

```diff
@@ -1971,6 +1999,14 @@ def validate_github(self, pr_number):
             rc = yield self.fail_build("PR {} is a draft pull request".format(pr_number))
             return defer.returnValue(rc)
 
+        commit_data = yield self.get_pr_commits(pr_number, repository_url, retry=3)
+        commit_msg = commit_data[0].get('commit', {}).get('message', '')
+        for line in commit_msg.lower().split():
+            if line.startswith('unreviewed'):
```

Comment:
I'm wondering if the way the commit message is currently checked is a better idea here?

Reply:
Could you please elaborate on "the way the commit message is currently checked"?
We grep for 'Unreviewed' in `ValidateCommitMessage` which is essentially equivalent to this method.

Reply:
Yes, happy to clarify. My comment was about the use of `startswith` specifically. It's probably not hugely important, just a thought since we were discussing the grep.

Reply:
To follow the spirit of `ValidateCommitMessage`, I'll change it to searching the entire message for 'Unreviewed' (case sensitive).